src/NodaTime.Demo/OffsetDemo.cs

```diff
@@ -62,6 +62,19 @@ public void ConstructionFromTimeSpan()
             Assert.AreEqual(5400, offset.Seconds);
         }
 
+        [Test]
+        public void Max()
+        {
+            var offset = Offset.FromSeconds(10);
+            var offset2 = Offset.FromSeconds(20);
+            var expected = Offset.FromSeconds(20);
```

Comment:
Rather than "expected" and "actual", I'd probably use "result" for the second variable, and not use the first at all - just inline that into the Assert.AreEqual part.